meta-resin-common/recipes-support/resinhup/resinhup/run-resinhup.sh

```diff
@@ -48,6 +49,7 @@ Options:
   --supervisor-tag <SUPERVISOR TAG>
         In the case of a successful host OS update, bring in a newer supervisor too
         using this tag.
+        Don't omit the 'v' in front of the version. e.g.: v1.2.3 and not 1.2.3.
 
```

Comment:
I agree. This is the current state of how this are in the updater. We can fix them with another PR. This only clarifies what you need to do when you run it.